### ISSUE TITLE
+mixin style include results in "TypeError: Cannot read property 'first' of undefined"

### ISSUE THREAD

**Issue by dotdat** [sasstools/sass-lint#380]
When using Sass syntax mixin includes (+foobar), extends-before-declarations.js expects these to have an atkeyword node and then attempts access properties on an undefined object resulting in the following error. This provides test cases that were able to reproduce the error and a fix to resolve it.

```
/.../node_modules/sass-lint/lib/rules/extends-before-declarations.js:18
          if (item.first('atkeyword').first('ident').content === 'extend') {
                                     ^
```
TypeError: Cannot read property 'first' of undefined

I've managed to fix this issue locally by checking first to see if time.first('atkeyword') produces anything at all:

```
// (lib/rules/extends-before-declarations.js:18)
if ((item.type === 'include' || item.type === 'extend')) {
  if (item.first('atkeyword') &&
    item.first('atkeyword').first('ident').content === 'extend') {
````

This is all using sass-lint 1.3.2

Edit:
Could possibly be related to this: https://github.com/tonyganch/gonzales-pe/issues/112
but it's hard to tell as there's not a whole lot of info to go on aside from the title

**Comment by DanPurdy** [comment]
Hi @dotdat 

Thanks for this, Sorry to have you close the other PR but this just helps us keep track of what's going on and also prevents us having repeated and multiple issues being opened (to a point).

Id be happy for you to open a new PR with the [DCO](https://github.com/sasstools/sass-lint/blob/develop/CONTRIBUTING.md#developer-certificate-of-origin) attached.

Thanks again for taking the time here.

**Comment by DanPurdy** [comment]
this was resolved in #422
